tests/wasm_test/src/lib.rs

```diff
@@ -203,6 +204,7 @@ fn run(is_windows: bool) -> std::io::Result<()> {
 
   // just ensure these don't panic
   assert!(sys.env_home_dir().is_some());
+  assert!(sys.env_programs_dir().is_some());
```

Comment:
There's no Wasm implemenation so this is failing. Maybe we should remove this or implement something somewhat naive like what env_home_dir() does? I'm guessing it would use the `LOCALAPPDATA` env var then add `\Programs` to it.

Reply:
The latter was my solution first, then I remembered about the previous discussion on the feature request issue so I didn't include it. I'll implement this soon.

Reply:
Resolved. On platforms other than Windows, it returns `None`. The assert also expects `Some` only when the platform is Windows.